config/locales/translations/zendesk_apps_support.yml

```diff
@@ -319,6 +324,11 @@ parts:
       key: "txt.apps.admin.error.app_build.translation.products_do_not_match_manifest_products"
       title: "App builder job: products specified in translation file don't match products in manifest"
       value: "Products in manifest (%{manifest_products}) do not match products in translations (%{translation_products})"
+  - translation:
+      key: "txt.apps.admin.error.app_build.translation.insecure_token_parameter_in_manifest"
+      title: "Validation message to indicate missing secure(true) field in manifest's token parameter"
+      value: "Make sure to set secure to true when using keys in Settings. Learn more: %{link}"
```

Comment:
I feel `secure` and `true` is also same as above? Is this a parameter of Manifest as well? If so, we want to add following to the title.

> secure and true are always English as used in manifest file: https://developer.zendesk.com/apps/docs/developer-guide/manifest

Reply:
Oh yes. I will add the link. I feel that I should use this link instead / as well

https://developer.zendesk.com/apps/docs/developer-guide/using_sdk#using-secure-settings

Reply:
Please use the link you posted, thanks!

Reply:
Thank you, Hiro. I have updated the title to be
```
   title: "Validation message to indicate missing secure(true) field in manifest's token parameter. Secure(true) in manifest refers to https://developer.zendesk.com/apps/docs/developer-guide/using_sdk#using-secure-settings"
```

Reply:
Can you also add following to the title please?

> secure and true are always English. Do not translate.

Reply:
Sure, @hiroiguchi . I will add that in. Thank you for the review :)